dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletFactory.java

```diff
@@ -361,4 +361,10 @@
      */
 	public abstract List<Map<String, String>> getMostViewedContent(String structureInode,Date startDate, Date endDate, User user) throws DotDataException;
 
+    protected List<Contentlet> findPageContentlets(String HTMLPageIdentifier, String containerId, String uniqueId, String orderby,
+            boolean working, long languageId) throws DotDataException, DotStateException, DotSecurityException {
+        // TODO Auto-generated method stub
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)